dotCMS/src/integration-test/java/com/dotcms/util/pagination/ContentTypesPaginatorTest.java

```diff
@@ -121,6 +126,43 @@ public void test_getItems_WhenFilterContainsContentTypeName_ReturnsTheContentTyp
         }
     }
 
+    /**
+     * Method to Test: {@link ContentTypesPaginator#getItems(User, String, int, int, String, OrderDirection, Map)}
+     * When: When the current indices are desactive or deleted
+     * Should: Should return NA in the entries property
+     *
+     * @throws DotSecurityException
+     * @throws DotDataException
+     */
+    @Test
+    public void whenTheIndicesAreDeactivateShouldReturnNAInEntries() throws DotDataException, IOException {
+        final IndiciesInfo indiciesInfo = APILocator.getIndiciesAPI().loadIndicies();
+
+        final String live = indiciesInfo.getLive();
+        final String working = indiciesInfo.getWorking();
+        try {
+            if (live != null) {
+                APILocator.getContentletIndexAPI().deactivateIndex(live.substring(live.indexOf(".") + 1));
+            }
+
+            APILocator.getContentletIndexAPI().deactivateIndex(working.substring(working.indexOf(".") + 1));
```

Comment:
Codacy found an issue: [String.indexOf(char) is faster than String.indexOf(String).](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5857184)